pep/app/pods/components/tables/cell/actions/component.ts

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+
+interface TablesCellActionsArgs {}
```

Comment:
fine for now, but we probably should have a base interface that has `rowValue`, `cellValue`, etc args defined (possibly in our table addon), that we can extend from for any cell components like this.